### PR TITLE
Update/ag 10 uds content

### PR DESCRIPTION
**Pull request by czgraggen**
## Overview

> Describe your change in detail

**Forum Issue #** `telus/technology-forum#0`
> Link to the [Technology Forum][technology-forum] where this issue was discussed _(optional)_.

### Details

> e.g.
> - add something to existing topic
> - create new topic
> - adjust grammar & typos

---

#### Meta

> Please read and confirm each of the following:

- [ ] this topic was discussed in the [Technology Forum][technology-forum] _(ignore if the pull request represents small changes)_
- [ ] provided a descriptive topic and overview of contribution
- [ ] documentation format follows the [topic template][template]
- [ ] branch is up to date with the `main` branch
- [ ] "work in progress" commits are squashed _(Hint: ["Squashing Commits"][guide-squash])_
- [ ] commits follow the [Conventional ChangeLog][conventional-changelog] format
- [ ] no sensitive content included, such as:
  - content considered competitive intelligence
  - security & privacy policy violating content
  - keys, tokens or credentials

[template]: https://github.com/telus/reference-architecture/blob/master/.template.md
[conventional-changelog]: https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-angular
[guide-squash]: https://git-scm.com/book/id/v2/Git-Tools-Rewriting-History
[technology-forum]: https://github.com/telus/technology-forum]

### CODE REVIEW

File: development/allium.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+- **Single platform, single brand** - Teams building applications on a single platform such as Web or mobile, that only support a single brand (TELUS) will use Allium.
+
+- **Multi-platform** - Teams building applications that will run on multiple platforms can use some Allium components and all UDS components to support cross-platform components. They may adopt UDS components directly and collaborate with the UDS team to customize them.
+
+- **Multi-brand** - Teams building applications that cut across several brands will need to adopt UDS components directly and tailor them as needed.
+
+## How should Allium be used?
+
+- For designers and content practitioners, Allium is available as a collection of UI libraries in Figma.
+- For developers, Allium is available as a single npm package, @telus-uds/ds-allium, for integration in consuming application.
```
Comment (jcayne-telus) [inline development/allium.md:41]:
```suggestion
- For developers, Allium is available as a single npm package, `@telus-uds/ds-allium`, for integration in consuming application.
```

File: development/allium.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+If you are working with Allium, reach out on slack at #ds-allium-support.
```
Comment (jcayne-telus) [inline development/allium.md:43]:
```suggestion
If you are working with Allium, reach out on Slack at #ds-allium-support.
```

Comment (jcayne-telus) [inline development/allium.md:43]: Should we include a link to the Slack channel(s) listed on https://telus.github.io/allium-design-system/contact or point the user to this page directly?

File: development/allium.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+## References
+
+- [Allium website](https://telus.github.io/allium-design-system/)
+- [Allium repo](https://github.com/telus/allium-design-system)
```
Comment (jcayne-telus) [inline development/allium.md:48]:
```suggestion
- [Allium GitHub repository](https://github.com/telus/allium-design-system)
```

File: development/css.md [hunk 4]
```diff
@@ -23,13 +23,15 @@ We currently use [styled-components](https://www.styled-components.com/) as our
 We also have some best practices to follow when styling applications:
 
 - Must use [style objects](https://www.styled-components.com/docs/advanced#style-objects) with the `styled` syntax when creating styled-components
-- Should try to reuse existing components (such as TDS components) before creating new styled-components
+- Should try to reuse existing components (such as [Allium](https://github.com/telus/allium-design-system) components) before creating new styled-components
```
Comment (jcayne-telus) [inline development/css.md:26]: Should we mention Allium is for TELUS branded components or reference UDS instead?

On the `development/React` page, should we replace TDS with UDS?

File: development/css.md [hunk 5]
```diff
@@ -23,13 +23,15 @@ We currently use [styled-components](https://www.styled-components.com/) as our
...
 - Recommend that functionality such as layouts and animations be created and shared via components
 - Must [sanitize](https://www.styled-components.com/docs/advanced#security) user input if it is to be used as styles
 
 ## References
 
 - [Mozilla Developer Network: CSS Reference](https://developer.mozilla.org/en-US/docs/Web/CSS/Reference)
+- [Universal Design System (UDS)](https://github.com/telus/universal-design-system)
+- [Monorepo for the UDS family of design systems](https://telus.github.io/universal-design-system/)
```
Comment (jcayne-telus) [inline development/css.md:34]:
```suggestion
- [Monorepo for the UDS family of design systems](https://github.com/telus/universal-design-system)
```

File: development/css.md [hunk 6]
```diff
@@ -23,13 +23,15 @@ We currently use [styled-components](https://www.styled-components.com/) as our
...
 - Must use [style objects](https://www.styled-components.com/docs/advanced#style-objects) with the `styled` syntax when creating styled-components
-- Should try to reuse existing components (such as TDS components) before creating new styled-components
+- Should try to reuse existing components (such as [Allium](https://github.com/telus/allium-design-system) components) before creating new styled-components
 - Recommend that functionality such as layouts and animations be created and shared via components
 - Must [sanitize](https://www.styled-components.com/docs/advanced#security) user input if it is to be used as styles
 
 ## References
 
 - [Mozilla Developer Network: CSS Reference](https://developer.mozilla.org/en-US/docs/Web/CSS/Reference)
+- [Universal Design System (UDS)](https://github.com/telus/universal-design-system)
```
Comment (jcayne-telus) [inline development/css.md:33]:
```suggestion
- [Universal Design System (UDS)](https://www.telus.com/uds)
```

Comment (jcayne-telus) [reply development/css.md:33]: Alternatively, can use https://telus.github.io/universal-design-system

File: development/uds.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
+# UDS
```
Comment (jcayne-telus) [inline development/uds.md:1]:
```suggestion
# Universal Design System (UDS)
```

File: development/uds.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+The Universal Design System (UDS) was created to support multiple TELUS owned businesses and initiatives. It is a design system that enables multi-branded, multi-platform experiences.
+
+### Benefits
+
+The benefits of UDS are:
+
+- **Brand-agnostic** - It includes the brand-agnostic React Native Base components
+- **Cross-platform** - React Native allows UDS Base components to be shared and reused on the Web, and in iOS and Android applications.
+- **Reusable** - The Base component library is a collection of UI elements that can be reused across multiple platforms and brands.
+- **Multi brand** - UDS Base components have a sophisticated theming mechanism which supports usage across multiple TELUS brands. Also components can be customized to support non-TELUS brands.
```
Comment (jcayne-telus) [inline development/uds.md:14]:
```suggestion
- **Multi-brand** - UDS Base components have a sophisticated theming mechanism which supports usage across multiple TELUS brands. Also components can be customized to support non-TELUS brands.
```

File: development/uds.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+- **Onmi channel** - UDS Base components can also vary in appearance according to the context or channel within a brand, allowing further degrees of reuse and flexibility.
```
Comment (jcayne-telus) [inline development/uds.md:15]:
```suggestion
- **Omni-channel** - UDS Base components can also vary in appearance according to the context or channel within a brand, allowing further degrees of reuse and flexibility.
```

File: development/uds.md [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+- **Multi-platform** - Teams building applications that will run on multiple platforms can use some Allium components and all UDS components to support cross-platform components. They may adopt UDS components directly and collaborate with the UDS team to customize them.
+- **Multi-brand** - Teams building applications that cut across several brands will need to adopt UDS components directly and tailor them as needed.
+
+## How should UDS be used?
+
+- **Cross-platform** - React Native allows UDS Base components to be shared and reused on the Web, as well as in iOS and Android applications.
+- **Multi-brand** - UDS Base components boast TELUS's sophisticated theming mechanism which enables them to support usage across multiple brands, including TELUS, Koodo and Public Mobile.
+- **Omni-channel** - UDS Base components can vary in appearance according to the context or channel within a brand, allowing further degrees of reuse and flexibility.
+
+If you are working with UDS, you can post your questions on #ux-architecture-community.
```
Comment (jcayne-telus) [inline development/uds.md:42]: Should we include a link to the Slack channel(s) listed on https://telus.github.io/allium-design-system/contact or point the user to this page directly?

File: development/uds.md [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+## Who are the primary users?
+
+The primary users are business teams outside of the TELUS brand, and multi-platform TELUS teams that are building or maintaining a design system or component library for use in the TELUS organization.
+
+## When should UDS be used?
+
+- **Multi-platform** - Teams building applications that will run on multiple platforms can use some Allium components and all UDS components to support cross-platform components. They may adopt UDS components directly and collaborate with the UDS team to customize them.
+- **Multi-brand** - Teams building applications that cut across several brands will need to adopt UDS components directly and tailor them as needed.
+
+## How should UDS be used?
```
Comment (jcayne-telus) [inline development/uds.md:36]: This section and the previous appear to be similar to the benefits section.

File: development/uds.md [hunk 12]
```diff
@@ -0,0 +1,47 @@
...
+- **Multi brand** - UDS Base components have a sophisticated theming mechanism which supports usage across multiple TELUS brands. Also components can be customized to support non-TELUS brands.
+- **Onmi channel** - UDS Base components can also vary in appearance according to the context or channel within a brand, allowing further degrees of reuse and flexibility.
+
+## What is UDS?
+
+UDS is a family of design systems, component libraries and supporting resources, which enable the accelerated development of consistent digital experiences across platforms, brands and business channels.
+
+UDS is the foundation for building design systems at TELUS, and supports the development of cross-platform and multi-brand applications.
+
+- UDS accelerates the creation of consistent user interfaces across all brands, platforms, and business channels at TELUS.
```
Comment (jcayne-telus) [inline development/uds.md:23]: Should the whole section be a list?

File: development/uds.md [hunk 13]
```diff
@@ -0,0 +1,47 @@
...
+
+- **Cross-platform** - React Native allows UDS Base components to be shared and reused on the Web, as well as in iOS and Android applications.
+- **Multi-brand** - UDS Base components boast TELUS's sophisticated theming mechanism which enables them to support usage across multiple brands, including TELUS, Koodo and Public Mobile.
+- **Omni-channel** - UDS Base components can vary in appearance according to the context or channel within a brand, allowing further degrees of reuse and flexibility.
+
+If you are working with UDS, you can post your questions on #ux-architecture-community.
+
+## References
+
+- [UDS website](https://telus.github.io/universal-design-system/)
```
Comment (jcayne-telus) [inline development/uds.md:46]: Could also be https://www.telus.com/uds, if used above.

File: development/uds.md [hunk 14]
```diff
@@ -0,0 +1,47 @@
...
+- [UDS repo](https://github.com/telus/universal-design-system)
```
Comment (jcayne-telus) [inline development/uds.md:47]:
```suggestion
- [UDS GitHub repository](https://github.com/telus/universal-design-system)
```

**Review by jcayne-telus — COMMENTED** [review]
Great addition for UDS!

Should the new entries be added to the Development section's table of contents (Readme)?

**Review by czgraggen — COMMENTED** [review]
Completed all changes requested by J Cayne

**Review by pdufault — APPROVED** [review]

**Review by gonzalovazquez — APPROVED** [review]

**Pull request merged**